docs/src/pages/quasar-cli/vuex-store.md

```diff
@@ -19,7 +19,7 @@ We won't go into details on how to configure or use Vuex since it has great docs
 
 By default, if you choose to use Vuex when you create a project folder with Quasar CLI, it will set you up on using Vuex modules. Each sub-folder of `/src/store` represents a Vuex Module.
 
-If by contrast, you did not choose to use Vuex when you create a project folder with Quasar CLI, Adding a Vuex Module (see below) is all you need to do to `install` vuex.
+If by contrast you did not choose Vuex option during the project creation, adding a Vuex Module (see below) is all you need to install Vuex.
```

Comment:
I would suggest
"If by contrast you did not choose the Vuex option during project creation, adding a Vuex Module (see below) is all you need in order to install Vuex."

Reply:
The "by contrast" sounded strange, had a native speaker help me to rephrase it.
"in order to"/"to do to ..." etc are most of the time just superfluous words which don't carry semantic meaning, that's why I removed them.
In general concise phrases are easier to understand

Reply:
For more references on the topic: https://v3.vuejs.org/guide/contributing/writing-guide.html#principles